plugins/metadata/dc11/filter/Dc11SchemaArticleAdapter.inc.php

```diff
@@ -209,16 +218,19 @@ public function &extractMetadataFromDataObject(&$article)
         }
 
         // Public identifiers
-        $pubIdPlugins = (array) PluginRegistry::loadCategory('pubIds', true, $journal->getId());
-        foreach ($pubIdPlugins as $pubIdPlugin) {
-            if ($issue && $pubIssueId = $issue->getStoredPubId($pubIdPlugin->getPubIdType())) {
+        $publicIdentifiers = [
+            'doi',
```

Comment:
This would always include DOIs even if they aren't enabled. If you want to first check if they're enabled before adding them to the array, `$journal->getData(Context::SETTING_ENABLE_DOIS)` would indicate whether they're enabled or not.

Reply:
I thought there could be something like that, as it was a plugin before (on/off switch), that's why I invited you :)
Thanks!